### ISSUE TITLE
Is this Repository Still Maintained?

### ISSUE THREAD

**Issue by 360disrupt** [aheckmann/gridfs-stream#128]
Found this [fork](https://github.com/HolgerFrank/gridfs-stream) which has some bugs fixed.

**Comment by jlyonsmith** [comment]
I don't think so.  Which is a problem because lots of folks depend on it, including me!  I'm going to at least fix #126 in [my fork](https://github.com/jlyonsmith/gridfs-stream) which is a show-stopper issue right now.

**Comment by florianeichin** [comment]
- Is there an alternative as of working with mongoose? 
- Is there any possibility to to donate this codebase to some other maintainers? As it would be missleading, if everyone starts forking this now.

**Comment by jlyonsmith** [comment]
@aheckmann I would be willing to take on maintenance and merging of pull requests for this code base.

**Comment by florianeichin** [comment]
I think this repository should get a deprecation warning and a hint to use the mongodbs official streaming api. Everyone using this repo/npm package should upgrade their code according to [this tutorial](http://mongodb.github.io/node-mongodb-native/2.1/tutorials/gridfs/streaming/)

**Comment by jlyonsmith** [comment]
@florianeichin I agree.  I have also converted over to the official MongoDB streaming API.

**Comment by piavgh** [comment]
@florianeichin , @jlyonsmith : do you guys know how to use mongodb official GridFSBucket API to do this task https://github.com/HolgerFrank/gridfs-stream#accessing-file-metadata ?

**Comment by jlyonsmith** [comment]
It’s on the man page for GridFSBucket, http://mongodb.github.io/node-mongodb-native/3.5/api/GridFSBucket.html

You can set the metadata in the options argumante.

For more detailed help Stackoverflow would be your best bet.

Cheers,

John.
